### PR TITLE
Bring Speech API samples up to standard.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: speech/recognize.js [hunk 1]
```diff
@@ -216,7 +216,8 @@ function streamingRecognize (filename, encoding, sampleRateHertz, languageCode)
       encoding: encoding,
       sampleRateHertz: sampleRateHertz,
       languageCode: languageCode
-    }
+    },
+    interimResults: false // Try setting to true to get interim results
```
Comment (ace-n) [inline speech/recognize.js]: I assume these are meant for the user? (If so, we should clarify that.)

Comment (jmdobry) [inline speech/recognize.js]: what would you change it to?

Comment (ace-n) [inline speech/recognize.js]: // If you want interim results, set this to true

Comment (jmdobry) [inline speech/recognize.js]: Done

**Pull request merged**